showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -44,14 +46,53 @@ class ShowkaseProcessor @JvmOverloads constructor(
     private val logger = ShowkaseExceptionLogger()
     private val showkaseValidator = ShowkaseValidator()
 
-    override fun getSupportedAnnotationTypes(): MutableSet<String> = mutableSetOf(
-        ShowkaseComposable::class.java.name,
-        PREVIEW_CLASS_NAME,
-        ShowkaseColor::class.java.name,
-        ShowkaseTypography::class.java.name,
-        ShowkaseRoot::class.java.name,
-        ShowkaseScreenshot::class.java.name,
-    )
+    override fun getSupportedAnnotationTypes(): MutableSet<String> = supportedAnnotationTypes()
+
+    private fun supportedAnnotationTypes(): MutableSet<String> {
+        val set = mutableSetOf(
+            ShowkaseComposable::class.java.name,
+            PREVIEW_CLASS_NAME,
+            ShowkaseColor::class.java.name,
+            ShowkaseTypography::class.java.name,
+            ShowkaseRoot::class.java.name,
+            ShowkaseScreenshot::class.java.name,
+        )
+
+        environment.getTypeElementsFromPackage(CODEGEN_PACKAGE_NAME)
```

Comment:
hmm I'm not sure if we should be doing this here 🤔

Reply:
hmm, yeah that is a good point actually 🤔 Wouldn't that always be the case for incremental builds though if I understand this correctly?

Reply:
I think what you need here is the custom multi preview annotation name instead of the custom annotation that we've created for codegen purposes

Reply:
Removed this and added the multi preview name from the options :)